docs/warnings.rst

```diff
@@ -34,20 +34,20 @@ for simple call `warnings.simplefilter`::
 
     >>> warnings.simplefilter('ignore', UserWarning)
 
-Astropy includes its own warning class,
-`~astropy.utils.exceptions.AstropyUserWarning`, on which all warnings from
-Astropy are based.  So one can also ignore warnings from Astropy (while still
-allowing through warnings from other libraries like Numpy) by using something
-like::
+Astropy includes its own warning classes,
+`~astropy.utils.exceptions.AstropyWarning` and `~astropy.utils.exceptions.AstropyUserWarning`,
+on which all warnings from Astropy are based.  So one can also ignore warnings
+from Astropy (while still allowing through warnings from other libraries
+like Numpy) by using something like::
 
-    >>> from astropy.utils.exceptions import AstropyUserWarning
-    >>> warnings.simplefilter('ignore', category=AstropyUserWarning)
+    >>> from astropy.utils.exceptions import AstropyWarning
+    >>> warnings.simplefilter('ignore', category=AstropyWarning)
```

Comment:
I think this could use better clarification (and perhaps I need to go through the source and review how warnings have actually been used).  `AstropyWarning` is indeed the base class for all astropy warnings, so this is accurate insofar as one can ignore all warnings from Astropy by using `AstropyWarning`.  However, I think there is a distinction worth making, that `AstropyUserWarning` is for warnings *intended* for general software users (e.g. "Warning: Ambiguous unit string" or something like that.  However, there are also things like `AstropyDeprecationWarning` which I think fall under a different category.  It's also useful to users using the library directly to know that they might not want to get accustomed to using some deprecated function.  But authors of software using Astropy should make this distinction more carefully perhaps (they might want to let through UserWarnings, while filter out Deprecation warnings which should not be seen by their users).